upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -908,27 +911,21 @@ func getELBTags(c AWSCloud, loadBalancerName string) (map[string]string, error)
 	request := &elb.DescribeTagsInput{
 		LoadBalancerNames: []*string{&loadBalancerName},
 	}
+	response, err := c.ELB().DescribeTags(request)
+	if err != nil {
+		return nil, fmt.Errorf("error listing tags on %v: %v", loadBalancerName, err)
+	}
 
-	attempt := 0
```

Comment:
I am confused that this function of for{} , 
this seems run one,so i think it is ok to remove this code .